src/components/Calculator.js

```diff
@@ -26,10 +26,13 @@ function Calculator() {
   const keys = keysElements(clickHandler);
   const { next, total } = calculatorData;
   return (
-    <div className="calculatorContainer">
-      <input type="text" className="result" value={next || total || 0} disabled />
-      {keys}
-    </div>
+    <>
```

Comment:
- [ ] @jaferIdrees great work on your design, I found your layout on this page doesn't match the layout given, below is a comparison

`Your Design`

![image](https://user-images.githubusercontent.com/61408860/183738331-6b9cd31b-fa8a-432b-bb94-c43bf191f220.png)


` Design given `

![image](https://user-images.githubusercontent.com/61408860/183738238-f526f7c3-cc68-4e65-9e1f-cc07605f8b30.png)